utils/scarb-macro-interface/Cargo.toml

```
@@ -0,0 +1,19 @@
+[package]
+name = "scarb-macro-interface"
```

Didn't we want this to be within `cairo-lang-` namespace?

```suggestion
name = "cairo-lang-macro-interface"
```

---

I just have not yet discussed this with the compiler team, so wasn't sure if we can use it from the get go 😅